# Description: On builds without neon, or where neon is option, we need to
#  include memset.arm.S in the build
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/skia/skia_library_opts.gyp b/skia/skia_library_opts.gyp
--- a/skia/skia_library_opts.gyp
+++ b/skia/skia_library_opts.gyp
@@ -93,17 +93,17 @@
           ],
           'cflags': [
             '-fomit-frame-pointer',
           ],
           'sources': [
             '../third_party/skia/src/opts/SkBitmapProcState_opts_arm.cpp',
           ],
         }],
-        [ 'target_arch == "arm" and (arm_version < 7 or (arm_neon == 0 and arm_neon_optional == 1))', {
+        [ 'target_arch == "arm" and (arm_version < 7 or arm_neon == 0)', {
           'sources': [
             '../third_party/skia/src/opts/memset.arm.S',
           ],
         }],
         [ 'target_arch == "arm" and arm_version < 6', {
           'sources': [
             '../third_party/skia/src/opts/SkBlitMask_opts_none.cpp',
             '../third_party/skia/src/opts/SkBlitRow_opts_none.cpp',
